client/blocks/image-editor/style.scss

```diff
@@ -83,7 +93,7 @@
 	border: 2px solid $white;
 	background: darken($gray, 52%);
 
-	&:before {
+	&::before {
```

Comment:
The linter was complaining, and I didn't stop to think if I was looking a`pseudo:class` or `pseudo:element`. Reverting.